portal-frontend/src/app/features/edit-submission/parcel-details/parcel-entry/parcel-entry.component.ts

```diff
@@ -156,6 +156,8 @@ export class ParcelEntryComponent implements OnInit {
 
       this.emitFormChangeOnSearchActions();
     }
+
+    this.showErrors ? this.parcelForm.markAllAsTouched() : this.parcelForm.markAsTouched();
```

Comment:
if else will be better for this since this is not assignment

Reply:
Updated!